libfreerdp/crypto/test/TestFuzzCryptoBase64Decode.c

```diff
@@ -0,0 +1,10 @@
+#include <freerdp/crypto/crypto.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	int ntlmTokenLength = 0;
+	BYTE* ntlmTokenData = NULL;
+	crypto_base64_decode((char*)Data, (int)Size, &ntlmTokenData, &ntlmTokenLength);
```

Comment:
better limit the input size to `int`, this way you have an integer overflow (and invalid input)

Reply:
Done.